docs/reference/changelog.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+## [1.0.1] - yyyy-mm-dd
+
+### Changed
+
+* The core extensions have been upgraded to v1.1.4: Bid statistics and details, Location, Milestone documents, Process level title and description.
+* The documentation pages of all extensions have been moved to the new [Extension Explorer](https://extensions.open-contracting.org/).
+* A patched release package schema is now distributed with the profile.
+* The build process for the documentation has been improved.
+* A schema browser has been added.
+* The documentType codelist has been updated:
```

Comment:
Put these normative changes as the second bullet. Consider having "Normative changes" and "Non-normative changes" sections.

Reply:
I've split out normative and non-normative changes in https://github.com/open-contracting-extensions/public-private-partnerships/pull/210/commits/22abffb4348bc1bb3de2f50d20bbf42b3f0242c3